src/spec/lib/NsidcOpenSearchResponse_spec.js

```diff
@@ -104,207 +148,129 @@ describe('NsidcOpenSearchResponse', function () {
-            expect(results.getResults()[0].externalDataUrl.title).toEqual(externalDataUrl.title);
-            expect(results.getResults()[0].externalDataUrl.href).toEqual(externalDataUrl.href);
+            expect(results.getResults()[0].orderDataUrl.title).toEqual(orderData.title);
+            expect(results.getResults()[0].orderDataUrl.href).toEqual(orderData.href);
         });
 
-        it.skip('extracts multiple data format fields from the response', function () {
-            var results, fakeFeed, fakeFeedEntryContents, fakeFormats;
-
-            fakeFormats = ['HDF, format'];
-
-            fakeFeedEntryContents = [
-                '   <id>' + fakeUrl + '</id>',
-                '   <title>' + fakeTitle + '</title>',
-                '   <updated>' + fakeUpdated + '</updated>',
-                '   <summary>' + fakeSummary + '</summary>',
-                '   <georss:box>-90 -180 90 180</georss:box>',
-                '   <dc:date>' + fakeStart + '/' + fakeEnd + '</dc:date>',
-                '   <dc:date>2013-07-01/2013-07-31</dc:date>',
-                '   <author><name>' + fakeAuthor + '</name></author>',
-                '   <dif:Parameters><dif:Category>category</dif:Category><dif:Detailed_Variable>' +
-            fakeParameter + '</dif:Detailed_Variable></dif:Parameters>',
-                '   <dif:Parameters><dif:Category>category2</dif:Category><dif:Detailed_Variable>' +
-            fakeParameter + '</dif:Detailed_Variable></dif:Parameters>',
-                '   <dif:Keyword>' + fakeKeyword + '</dif:Keyword>',
-                '   <dif:Distribution><dif:Distribution_Format>' + fakeFormats + '</dif:Distribution_Format></dif:Distribution>'
-            ];
-            fakeFeed = fakeFeedTop.concat(fakeFeedEntryContents).concat(fakeFeedBottom).join('\n');
-
-            results = nsidcOpenSearchResponse.fromXml(fakeFeed, {osSearchTerms: '', osDtStart: '', osDtEnd: '', geoBoundingBox: ''});
-
-            expect(results.getResults()[0].dataFormats).toEqual(['HDF, format']);
+        it('extracts brokered data links', function () {
+            let externalData = {
+                title: 'Get External Data',
+                href: 'http://data.eol.ucar.edu/codiac/dss/id=106.arcss054/',
+                rel: 'external-data'
+            };
+            let fakeResponse = fakeResponseHeader + '<entry>' +
+              fakeResponseEntry()+
+              link(externalData) +
+              '</entry></feed>';
+            let results = nsidcOpenSearchResponse.fromXml(
+              fakeResponse, {osSearchTerms: '', osDtStart: '', osDtEnd: '', geoBoundingBox: ''}
+            );
+
+            expect(results.getResults()[0].externalDataUrl.title).toEqual(externalData.title);
+            expect(results.getResults()[0].externalDataUrl.href).toEqual(externalData.href);
         });
 
-        it.skip('extracts multiple temporal fields from the response', function () {
```

Comment:
This test appears to have been removed completely. Could be I'm just not seeing it, but if it was removed was that intentional?

Reply:
Yes, because (unless I'm missing something) it seems to be a testing the same thing as the test starting on line 107 (previously line 105).  I figured removing the duplicate was cleaner than trying to clean/fix it :)  But I can re-add it if you think they are different enough to warrant having both.